### PR TITLE
✨ NEW: Add logo_only theme option

### PR DESCRIPTION
**Pull request by djangoliv**
Support for logo_only option.

Regars

### CODE REVIEW

**Comment by choldgraf** [comment]
This looks good to me, with a couple extra requests:

- could you add documentation of this new feature somewhere around https://sphinx-book-theme.readthedocs.io/en/latest/configure.html#customize-the-logo-title-and-favicon ?
- could you install `pre-commit` and run `pre-commit install` before you make your next commit? This will ensure that the pre-commit tests pass

**Comment by djangoliv** [comment]
@choldgraf  done: 

pre-commit ok:
```
Check JSON...........................................(no files to check)Skipped
Check Yaml...........................................(no files to check)Skipped
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
flake8...............................................(no files to check)Skipped
black................................................(no files to check)Skipped
check-manifest...........................................................Passed
Compile Web Assets...................................(no files to check)Skipped



```

File: docs/configure.md [hunk 1]
```diff
@@ -218,6 +218,12 @@ html_favicon = "path/to/favicon.ico"
 These will be placed in the top-left of your page.
 
 
+You can avoid the display of the project name at the top of the sidebar (below the logo) by adding this line in `conf.py`:
```
Comment (choldgraf) [inline docs/configure.md:221]:
```suggestion
To **omit the site title** below the logo, add this line in `conf.py`:
```

File: sphinx_book_theme/theme.conf [hunk 2]
```diff
@@ -13,6 +13,7 @@ repository_url =
 repository_branch =
 launch_buttons = {}
 home_page_in_toc = False
+logo_only =
```
Comment (choldgraf) [inline sphinx_book_theme/theme.conf:16]:
```suggestion
logo_only =
```

File: sphinx_book_theme/theme.conf [hunk 3]
```diff
@@ -13,6 +13,7 @@ repository_url =
...
```
Comment (choldgraf) [inline sphinx_book_theme/theme.conf:16]:
```suggestion
logo_only = False
```

this is giving the pre-commit check some problems so let's just explicitly set it to False as we do this with other bools in the config

**Comment by choldgraf** [comment]
ahh I figured out what was going on - we needed to update the conf in `src/jinja` not the one in `sphinx_book_theme`

**Comment by choldgraf** [comment]
caught a little bug in the docs (we need this option provided under `html_theme_options`). Also added a test. Should be working now

**Pull request merged**